### PR TITLE
Various fixes for the market and some bug fixes

### PR DESCRIPTION
**Pull request by devos50**
In this PR, I fixed some small bugs and added a token overview to the homepage.

I've also changed some functionality of the market. In particular, an order is now only valid iff a witness has signed the ask/bid creation block.

### CODE REVIEW

**Comment by qstokkink** [comment]
```python
ERROR   1520860127            util:Tribler.dispersy.util:239  Exception raised on the reactor's thread AttributeError: "'NoneType' object has no attribute '_use_main_thread'".
 Traceback from the reactor's thread:
   File "/usr/lib/python2.7/dist-packages/twisted/internet/defer.py", line 1386, in _inlineCallbacks
    result = g.send(result)
  File "/home/jenkins/workspace/GH_Tribler_PR_tests_linux/tribler/Tribler/Test/Core/Modules/RestApi/test_trustchain_endpoint.py", line 17, in setUp
    self.session.lm.triblerchain_community._use_main_thread = False
```

**Comment by qstokkink** [comment]
retest this please

**Comment by devos50** [comment]
retest this please

File: Tribler/Core/Modules/restapi/market/asks_bids_endpoint.py [hunk 1]
```diff
@@ -101,8 +102,14 @@ def render_PUT(self, request):
             request.setResponseCode(http.BAD_REQUEST)
             return json.dumps({"error": "price_type or quantity_type parameter missing"})
 
-        self.get_market_community().create_ask(*BaseAsksBidsEndpoint.create_ask_bid_from_params(parameters))
-        return json.dumps({"created": True})
+        def on_ask_created(_):
+            request.write(json.dumps({"created": True}))
+            request.finish()
```
Comment (qstokkink) [inline Tribler/Core/Modules/restapi/market/asks_bids_endpoint.py]: If `request.write` failed, the request will already be finished. Please put this in an `if not request.finished:` statement.

**Pull request merged**